cobbler/api.py

```diff
@@ -46,6 +46,7 @@
 from cobbler.utils import input_converters, signatures, filesystem_helpers
 from cobbler.cobbler_collections import manager
 from cobbler.items import (
+    item,
```

Comment:
Same as before. Let's put that in a conditional please if it is only required for a type annotation.

Reply:
I tried to change the code like this:
```diff
-from cobbler.items import (
-    item,
-    distro,
-    file,
-    image,
-    menu,
-    mgmtclass,
-    package,
-    profile,
-    repo,
-    system,
-)
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from cobbler.items import (
+        item,
+        distro,
+        file,
+        image,
+        menu,
+        mgmtclass,
+        package,
+        profile,
+        repo,
+        system,
+    )
```
but got this error:
````
ImportError while loading conftest '/usr/src/cobbler/tests/conftest.py'.
../conftest.py:8: in <module>
    from cobbler.api import CobblerAPI
../../cobbler/api.py:73: in <module>
    class CobblerAPI:
../../cobbler/api.py:2033: in CobblerAPI
    def deserialize_item(self, obj: item.Item) -> dict:
E   NameError: name 'item' is not defined
````

Reply:
See above comment again. :)